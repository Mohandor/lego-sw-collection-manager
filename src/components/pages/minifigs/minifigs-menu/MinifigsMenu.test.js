import React from 'react';
import MinifigsMenu from './MinifigsMenu';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';


describe('<MinifigsMenu />', () => {
  it('should render the Minifigs Menu', () => {
    const wrapper = shallow(<MinifigsMenu />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});