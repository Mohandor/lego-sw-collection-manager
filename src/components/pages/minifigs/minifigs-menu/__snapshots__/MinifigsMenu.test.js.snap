// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MinifigsMenu /> should render the Minifigs Menu 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={6}
    xs={12}
  >
    <WithStyles(ForwardRef(Paper))
      classes={
        Object {
          "root": "Paper",
        }
      }
    >
      <WithStyles(ForwardRef(Grid))
        alignItems="center"
        className="Container"
        container={true}
      >
        <WithStyles(undefined) />
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Paper))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={6}
    xs={12}
  >
    <WithStyles(ForwardRef(Paper))
      classes={
        Object {
          "root": "Paper",
        }
      }
    >
      <WithStyles(ForwardRef(Grid))
        className="Container"
        container={true}
      >
        <Connect(filters) />
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Paper))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
